packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -626,9 +626,14 @@ class AndroidDevice extends Device {
       );
     }
 
-    List<String> cmd;
+    // TODO(jonahwilliams): remove after enabling null_assertions by default
```

Comment:
Is there an issue for doing this on file that could be linked here?

Reply:
Done